src/odemis/acq/test/move_test.py

```diff
@@ -49,15 +48,7 @@ class TestCryoMove(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        try:
-            test.start_backend(ENZEL_CONFIG)
-        except LookupError:
-            logging.info("A running backend is already found, skipping tests")
-            cls.backend_was_running = True
-            return
-        except IOError as exp:
-            logging.error(str(exp))
-            raise
+        test.start_backend(ENZEL_CONFIG)
```

Comment:
Just a note:
In case the correct back-end is already running this is logged by _src/odemis/util/test.py_, although not very explicit this should be enough for the user of this function I think.